### DISCUSSION TITLE
Use Music Assistant as DLNA Media server

### DISCUSSION THREAD

**Discussion by marcelveldt** [music-assistant/support#388] [Feature requests and ideas]
Implement a DLNA Media Server interface so that the media items within Music Assistant can be browsed from DLNA compatible renderers. Might also solve the issue with missing metadata on the player device itself

**Comment by busywait** [comment]
When I installed Music Assistant I thought that it *was* a DNLA renderer, and was initially disappointed to discover not. Now that I've used Music Assistant I appreciate the gapless playback (especially "Strict mode") to Chromecast devices because I've not been able to do that via DNLA. Also I now have the ability to skip through .flac files on my Yamaha MusicCast WXAD-10 when I use the Music Assistant player UI, which isn't possible via the native Yamaha controls at this time.

So I'm happy to give up the metadata on my Google Nest display (and being able to browse via the BubbleUPNP app) for the gapless playback. But having all of that together would be great :)

**Comment by sbelyea** [comment]
Adding some support for this request.  I've got some SqueezeBox Radio devices that will happy connect to a DLNA server; my use case would be to use Music Assistant to access my Apple Music account, and serve up playlists/access to those DLNA renderers.

**Reply by OzGav** [reply]
You should use the squeezelite provider to connect to these

**Reply by sbelyea** [reply]
Thanks @OzGav - I completely missed that capability when looking at the documents.  This solved my issue.

**Comment by OzGav** [comment]
We looked at implementing this but upon review decided that it introduces security risks which can’t be overcome at this time and thus this won’t be implemented without more demand and a way forward to make it secure.
